app/src/main/java/ai/elimu/filamu/ui/video/VideoActivity.kt

```diff
@@ -85,13 +88,14 @@ class VideoActivity : AppCompatActivity() {
                     Timber.tag(TAG).v("onPlaybackStateChanged: %s", playbackState)
 
                     if (playbackState == Player.STATE_ENDED) {
-                        val videoTitle = intent.getStringExtra(EXTRA_KEY_VIDEO_TITLE)
+                        videoTitle = intent.getStringExtra(EXTRA_KEY_VIDEO_TITLE) ?: ""
                         LearningEventUtil.reportVideoLearningEvent(
                             VideoGson().apply {
                                 id = videoId
                                 title = videoTitle
```

Comment:
@tuancoltech Would it be possible to also include the `additionalData` property here for collecting the current timestamp of the video player?

For example something like this?: https://github.com/elimu-ai/ml-event-simulator/blob/main/lang-VIE/android-id-e387e38700000001/version-code-3001018/video-learning-events/e387e38700000001_3001018_video-learning-events_2025-06-11.csv